mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -597,12 +768,13 @@
         <SubgroupPriority value="93"/>
         <WeaponArray Link="HighTemplarWeapon"/>
         <AbilArray Link="attack"/>
+        <Speed value="2.0156"/>
     </CUnit>
     <CUnit id="Hive">
         <Food value="6"/>
         <Collide index="Locust" value="1"/>
         <Collide index="Phased" value="1"/>
-        <SubgroupPriority value="28"/>
+        <SubgroupPriority value="32"/>
```

Comment:
**undocumented**: Hatchery and Hive's subgroup priority swapped (Hatchery to 28, Hive to 32)

This changes the priority of the wireframe or unit button in several UI elements for observers, but most important is the multi-unit-selection panel showing all the wireframes